doc/reference/lora_lorawan/index.rst

```diff
@@ -0,0 +1,92 @@
+.. _lora_api:
+.. _lorawan_api:
+
+LoRa and LoRaWAN
+################
+
+Overview
+********
+
+LoRa (abbrev. for Long Range) is a proprietary low-power wireless
+communication protocol developed by the `Semtech Corporation`_.
+
+LoRa acts as the physical layer (PHY) based on the chirp spread spectrum
+(CSS) modulation technique.
+
+LoRaWAN (for Long Range Wide Area Network) defines a networking layer
```

Comment:
Perhaps `(for LoRa Wide Area Network)` instead of duplicating the `Long Range` part